test/src/main/java/org/corfudb/test/LsofSpec.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.test;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.fail;
+
+@Slf4j
+public class LsofSpec {
```

Comment:
I think a more general approach for this spec should filter on `'(deleted)'` and not the segment path. That way it will find all deleted files with an open file descriptor.

Reply:
The problem that `lsof` output is different on Mac and Linux. Mac version doesn't have '(deleted)' in its output.